src/main/CommandHandler.cpp

```diff
@@ -277,6 +278,8 @@ CommandHandler::fileNotFound(std::string const& params, std::string& retStr)
         "</p><p><h1> /metrics</h1>"
         "returns a snapshot of the metrics registry (for monitoring and "
         "debugging purpose)"
+        "</p><p><h1> /clearmetrics?[domain=DOMAIN]</h1>"
+        "clear metrics for all/specified domains (for testing purposes)"
```

Comment:
the same string is also in `docs/software/commands.md`

Reply:
I opened https://github.com/stellar/stellar-core/issues/1589 to avoid having to deal with this problem in the future

Reply:
you also need to document the behavior if domain is not specified